allensdk/brain_observatory/ecephys/ecephys_session.py

```diff
@@ -881,6 +988,11 @@ def from_nwb_path(cls, path, nwb_version=2, api_kwargs=None, **kwargs):
 
         return cls(api=NWBAdaptorCls.from_path(path=path, **api_kwargs), **kwargs)
 
+    def _warn_invalid_spike_intervals(self):
```

Comment:
you are checking all invalid time intervals here. is it possible for lfp or stim to be invalid while spikes are OK?

Reply:
Good point @NileGraddis. I have limited warning to "all_probes" and "probe#" tags now.
@jsiegle, could you please comment which of the failing tags affect spikes?

Reply:
All the physiology (LFP and spikes) will be invalid simultaneously. But the spikes and LFP are both fine if the stimulus is invalid.

Reply:
thank @jsiegle, this agrees with the most recent implementation.